x-pack/plugin/async-search/src/main/java/org/elasticsearch/xpack/search/TransportSubmitAsyncSearchAction.java

```diff
@@ -126,7 +126,10 @@ public void onFailure(Exception exc) {
 
                 @Override
                 public void onFailure(Exception exc) {
-                    submitListener.onFailure(exc);
+                    //this will only ever be called when there's an issue scheduling the thread will invoke
+                    //the completion listener once the wait for completion timeout expires
```

Comment:
Can you reword this comment to make it understandable ?

Reply:
ops ok